redis/index.js

```diff
@@ -74,7 +74,8 @@ function get(key) {
  * @return {[type]}
  */
 function batch(ops) {
-  var batch = [];
+  var batch = [],
```

Comment:
```suggestion
  const batch = [],
```

Reply:
Ah I was under the impression that `const` would make `batch` immutable, but that doesn't seem to be the case. I can change this.